src/main/ml-modules/base/root/config/searchTermConfig.mjs

```diff
@@ -73,6 +73,13 @@ const SEARCH_TERM_CONFIG = {
       hopInverseName: 'occupationOf',
       indexReferences: ['conceptPrimaryName'],
     },
+    professionalActivity: {

```

Comment:
This search term config doesn't look right to me. It shouldn't be a 'relatedList' pattern. I think it should be an 'indexedValue' similar to the `identifier` pattern below (just replace the `identifier` property name and `indexReference`)
```
    identifier: {
      patternName: 'indexedValue',
      indexReferences: ['referenceIdentifier'],
      scalarType: 'string',
    },
```